library/sbi-spec/src/binary.rs

```diff
@@ -22,32 +22,46 @@ pub struct SbiRet<T = usize> {
 }
 
 /// SBI success state return value.
+#[doc(alias = "sbi_ret_success")]
 pub const RET_SUCCESS: usize = <usize as SbiRegister>::RET_SUCCESS;
```

Comment:
```suggestion
pub const RET_SUCCESS: usize = <usize as SbiRegister>::RET_SUCCESS;
```

There's no such function named `sbi_ret_success`, it should not be marked.